extensions/store/media-source/test/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
What if you were to apply this to `extensions/.eslintrc.js` directly, so we don't run into the issue again in the future? I'm actually surprised this hasn't come up in the past.